frontend/components/settings/organisation/StripeBillingInfo.tsx

```diff
@@ -0,0 +1,354 @@
+
+  const handleCancelSubscription = async () =>
+    await cancelSubscription({
+      variables: { subscriptionId, organisationId: activeOrganisation!.id },
+      refetchQueries: [
+        { query: GetSubscriptionDetails, variables: { organisationId: activeOrganisation?.id } },
+        { query: GetOrganisations },
+      ],
+    }).then(() => {
+      toast.success('Cancelled subscription')
+      if (dialogRef.current) dialogRef.current.closeModal()
+    })
+
+  return (
+    <GenericDialog
+      title="Cancel subscription"
+      buttonContent={
+        <>
+          <FaTimes /> Cancel
+        </>
+      }
+      buttonVariant="danger"
+      ref={dialogRef}
+    >
+      <div>
+        <div className="py-4">
+          <p className="text-neutral-500">
+            Are you sure you want to cancel your subscription of Phase Pro? You will lose access to
+            all current and future Phase Pro features.
+          </p>
+        </div>
+        <div className="flex items-center justify-end">
+          <Button variant="danger" onClick={handleCancelSubscription}>
+            <FaTimes /> Cancel subscription
+          </Button>
+        </div>
+      </div>
+    </GenericDialog>
+  )
+}
+
+export const StripeBillingInfo = () => {
+  const { activeOrganisation } = useContext(organisationContext)
+
+  const { data, loading } = useQuery(GetSubscriptionDetails, {
+    variables: { organisationId: activeOrganisation?.id },
+    skip: !activeOrganisation,
+  })
+
+  const subscriptionData: StripeSubscriptionDetails | undefined =
+    data?.stripeSubscriptionDetails ?? undefined
+
+  const defaultPaymentMethod =
+    subscriptionData?.paymentMethods!.length === 1
+      ? subscriptionData?.paymentMethods[0]
+      : subscriptionData?.paymentMethods!.find((paymentMethod) => paymentMethod?.isDefault)
+
+  if (loading || !subscriptionData)
+    return (
+      <div className="flex items-center justify-center p-40 mx-auto">
+        <Spinner size="md" />
+      </div>
+    )
+
+  return (
+    <div className="space-y-6">
+      <div className="font-semibold text-xl">Current Subscription</div>
+      <div className="p-4 rounded-lg border border-neutral-500/40 border-t-8 border-t-emerald-500 bg-zinc-100 dark:bg-zinc-800">
+        <div className="font-medium pb-4">
+          {subscriptionData.planName}{' '}
+          <span className="capitalize">({subscriptionData.status})</span>
+        </div>
+        <div className="text-neutral-500 text-sm">
+          Current billing cycle:{' '}
+          {new Date(subscriptionData.currentPeriodStart! * 1000).toDateString()}
+          {' - '}
+          {new Date(subscriptionData.currentPeriodEnd! * 1000).toDateString()}
+        </div>
+
+        <div className="flex items-center justify-between">
+          <div className="text-neutral-500 text-sm">
+            Next payment {relativeTimeFromDates(new Date(subscriptionData.renewalDate! * 1000))}
```

Comment:
![image](https://github.com/user-attachments/assets/a2589cc7-9617-43e1-aede-b69e6817b045)
use bold text for the `relativeTimeFromDates` and `defaultPaymentMethod.last4`